corona-warn-app/cwa-app-android
# DccTicketing http logging modulized approach (EXPOSUREAPP-10738)

Addresses Exposureapp-10738.
Basing on Mtwalli's development (see previous PR).

How to test:
- active logging / report
- make some DccTicketing stuff / requests
- check the logs for relevant http entries
